src/Character.ts

```diff
@@ -8,84 +8,26 @@ import {
 } from './Util/PointUtils'
 
 export class Character {
-  readonly stats: Stats = new Stats()
-  readonly bonuses: Bonuses = new Bonuses()
-  private abilityPoints: number = 30
-  private level: number = 1
-  private isCreating: boolean = true
-  private trainingPoints: number = 0
+  readonly stats: Stats
+  readonly bonuses: Bonuses
+  private abilityPoints = 30
+  private trainingPoints = 0
+  private level = 1
+  private isCreating = true
 
-  //region add stats
-
-  addDexterity(dexterityToAdd: number): void {
-    if (dexterityToAdd > this.abilityPoints) {
-      throw new AbilityPointError('not enough points remaining')
-    }
-    this.abilityPoints -= dexterityToAdd
-    this.stats.dexterity += dexterityToAdd
-  }
-
-  addStrength(strengthToAdd: number): void {
-    if (strengthToAdd > this.abilityPoints) {
-      throw new AbilityPointError('not enough points remaining')
-    }
-    this.abilityPoints -= strengthToAdd
-    this.stats.strength += strengthToAdd
+  constructor() {
+    this.stats = new Stats(this)
```

Comment:
This is the tradeoff of not proxying; the `Stats` class needs to be aware of the callback for the `Character` class when updating stats so we just pass the whole character for now. Today the `Character` class still handles points, though that could certainly be pushed into the `Stats` class, but eventually we'll need to recalculate lots of things when underlying stats are updated.